src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -663,7 +663,7 @@ private void PreProcessRunStatement(ParseNode node)
                         currentCodeSection = subprogramObject.FunctionCode;
                         // verify if the program has been loaded
                         Opcode functionStart = AddOpcode(new OpcodePush(subprogramObject.PointerIdentifier));
-                        AddOpcode(new OpcodePush(0));
+                        AddOpcode(new OpcodePush(-1));
```

Comment:
I changed the dummy value to one that deliberately fails instead of 0, to help avoid the danger of jumping to the start of the program again like it did before.  If it is having the error of calling an uninitialized program address, I'd rather it crashed the program than recursively start itself over because it happens to jump to zero.